images/server/Dockerfile.fedora

```diff
@@ -13,7 +13,7 @@ MAINTAINER John Mulligan <jmulligan@redhat.com>
 ENV SAMBACC_VERSION="0.1"
 
 COPY smb.conf /etc/samba/smb.conf
-RUN dnf install -y \
+RUN dnf install --setopt=install_weak_deps=False -y \
```

Comment:
what's the reason for this added flag?

Reply:
We started pulling in more than we needed, while it wasn't much more I remembered that `--setopt=install_weak_deps=False ` is a good practice when  running dnf in containers.

Reply:
ah, cool, thanks. wasn't aware...